src/Microsoft.Azure.ServiceBus/Management/NamespaceInfo.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.ServiceBus.Management
+{
+    using System;
+
+    /// <summary>
+    /// Represents the metadata related to a service bus namespace.
+    /// </summary>
+    public class NamespaceInfo
+    {
+        /// <summary>
+        /// Name of the namespace.
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Type of entities present in the namespace.
+        /// </summary>
+        public NamespaceType NamespaceType { get; set; }
+
+        /// <summary>
+        /// The time at which the namespace was created.
+        /// </summary>
+        public DateTime CreatedTime { get; set; }
+
+        /// <summary>
+        /// The last time at which the namespace was modified.
+        /// </summary>
+        public DateTime ModifiedTime { get; set; }
+
+        /// <summary>
+        /// The SKU/tier of the namespace. Valid only for <see cref="NamespaceType.Messaging"/>
+        /// </summary>
+        public MessagingSku MessagingSku { get; set; }
+
+        /// <summary>
+        /// Number of messaging units allocated for namespace. 
+        /// Valid only for <see cref="NamespaceType.Messaging"/> and <see cref="MessagingSku.Premium"/>
+        /// </summary>
+        public int MessagingUnits { get; set; }
+
+        /// <summary>
+        /// Alias for the namespace.
+        /// </summary>
+        public string Alias { get; set; }
```

Comment:
Alias? That's new... is there a documentation on this?

Reply:
Still think it's a bit vague

Reply:
For now I'm going to leave it as-is since multiple changes might come up for this. Once clear documentation is released, will update the xml text here.

---
In reply to: [224308639](https://github.com/Azure/azure-service-bus-dotnet/pull/585#discussion_r224308639) [](ancestors = 224308639)